rasa/shared/core/events.py

```diff
@@ -1935,3 +1936,79 @@ def apply_to(self, tracker: "DialogueStateTracker") -> None:
         """Applies event to current conversation state."""
         # noinspection PyProtectedMember
         tracker._reset()
+
+
+class WronglyPredictedAction(ActionExecuted):
```

Comment:
I'm pretty sure we don't want to move these here. Once something is in `shared` it can be used by Rasa X and has to be supported accordingly. These are also test only events right? Not something that can actually be predicted.

Reply:
+1